scripts/webpack.conf.js

```diff
@@ -1,47 +1,95 @@
 const { modifyWebpackConfig, shallowMerge, defaultOptions } = require('@gera2ld/plaid');
+const { isProd } = require('@gera2ld/plaid/util');
 const webpack = require('webpack');
 const WrapperWebpackPlugin = require('wrapper-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
 const projectConfig = require('./plaid.conf');
+const mergedConfig = shallowMerge(defaultOptions, projectConfig);
 
 const INIT_FUNC_NAME = 'VMInitInjection';
 
-module.exports = Promise.all([
-  modifyWebpackConfig(async (config) => {
-    config.plugins.push(
-      new webpack.DefinePlugin({
-        'process.env.INIT_FUNC_NAME': JSON.stringify(INIT_FUNC_NAME),
-        'process.env.DEBUG': JSON.stringify(process.env.DEBUG || false),
-      }),
-    );
+const definitions = new webpack.DefinePlugin({
+  'process.env.INIT_FUNC_NAME': JSON.stringify(INIT_FUNC_NAME),
+  'process.env.DEBUG': JSON.stringify(process.env.DEBUG || false),
+});
+const minimizerOptions = {
+  cache: true,
+  parallel: true,
+  sourceMap: true,
+  terserOptions: {
+    output: {
+      ascii_only: true,
+    },
+  },
+};
+const minimizer = isProd && [
+  new TerserPlugin({
+    chunkFilter: ({ name }) => !name.includes('public/'),
+    ...minimizerOptions,
+    terserOptions: {
+      ...minimizerOptions.terserOptions,
+      compress: {
+        ecma: 6,
+        // 'safe' since we don't rely on function prototypes
+        unsafe_arrows: true,
+      },
+    },
+  }),
+  new TerserPlugin({
+    chunkFilter: ({ name }) => name.includes('public/'),
+    ...minimizerOptions,
+  }),
+];
+
+const modify = (extra, init) => modifyWebpackConfig(
+  (config) => {
+    config.plugins.push(definitions);
+    if (init) init(config);
     return config;
+  }, {
+    projectConfig: {
+      ...mergedConfig,
+      ...extra,
+      optimization: {
+        ...mergedConfig.optimization,
+        ...(extra || {}).pages && {
+          runtimeChunk: false,
+          splitChunks: false,
+        },
+        minimizer,
+      },
+    },
+  },
+);
+
+module.exports = Promise.all([
+  modify(),
+  modify({
+    pages: {
+      injected: {
+        entry: './src/injected',
+      },
```

Comment:
If not splitted, the generated `injected.js` will be a little smaller as I tested. Because it has to bundle a copy of runtime when splitted.

Reply:
I don't understand. In my tests it's the other way around: here `injected` is 11kB, when it's in the old place in plaid it's 14.5kB. What "runtime" does it have to bundle if there's an explicit `runtimeChunk: false` in the config?

Reply:
Sorry, I was wrong, it's bundled with a copy of common utils.
For me it's 10.33KB (non-splitted) vs 11.11KB (splitted).

Reply:
Sorry, I was still wrong. You are right about it. 😭

Reply:
I can't reproduce 10.33kB. If I restore the old plaid.conf.js and remove `modify` call for `entry: './src/injected'`, I get 14.5kB.